org.lflang/src/org/lflang/TargetProperty.java

```diff
@@ -349,6 +362,45 @@ private TargetProperty(String description, TargetPropertyType type,
         this.type = type;
         this.supportedBy = supportedBy;
         this.setter = setter;
+        this.updater = (config, value) -> { /* Ignore the update by default */ };
+    }
+    
+    /**
+     * Private constructor for target properties. This will take an additional
+     * `updater`, which will be used to merge target properties from imported resources.
+     * 
+     * @param description String representation of this property.
+     * @param type        The type that values assigned to this property
+     *                    should conform to.
+     * @param supportedBy List of targets that support this property.
+     * @param setter      Function for setting configuration values.
+     * @param updater     Function for updating configuration values.
+     */
+    private TargetProperty(String description, TargetPropertyType type,
+            List<Target> supportedBy,
+            BiConsumer<TargetConfig, Element> setter,
+            BiConsumer<TargetConfig, Element> updater) {
+        this.description = description;
```

Comment:
Why not just call the other constructor here?

Reply:
Ah nice suggestion.

Reply:
Unfortunately, `updater` being final means that we can't call the other constructor, which sets the `updater` to an empty lambda function.

Reply:
Ah, right.